src/x509.c

```diff
@@ -9228,6 +9466,21 @@ WOLF_STACK_OF(WOLFSSL_X509)* wolfSSL_X509_chain_up_ref(
                     }
                 }
             }
+
+        #ifdef WOLFSSL_CUSTOM_OID
+            if ((req->customExtCount < 0) ||
+                (req->customExtCount >= NUM_CUSTOM_EXT)) {
+                WOLFSSL_MSG("Bad value for customExtCount.");
+                ret = WOLFSSL_FAILURE;
```

Comment:
Setting ret here does not avoid the upcoming for loop and accessing cert->customCertExt.

Reply:
Good catch!!